en/components/avatar.md

```diff
@@ -105,7 +105,7 @@ Analogically, the avatar can display an icon via the [`icon`]({environment:angul
 ### API References
 <div class="divider--half"></div>
 
-* [IgxAvatarComponent]({environment:angularApiUrl}/classes/igxavatarcomponent.html)
+* [IgxAvatarComponent]({environment:sassApiUrl}/classes/igxavatarcomponent.html)
```

Comment:
Shouldn't this line reference the [IgxAvatar typedoc docs](https://staging.infragistics.local/products/ignite-ui-angular/docs/typescript/classes/igxavatarcomponent.html)?

Reply:
You are absolutely right. Fixed with the latest commit.